packages/hierarchies/src/hierarchies/HierarchyFiltering.ts

```diff
@@ -86,6 +88,8 @@ export interface FilteringPathAutoExpandDepthInHierarchy {
    * Then you provide `autoExpand: { depthInHierarchy: 2 }`
    *
    * To get the correct depth use `HierarchyNode.parentKeys.length`.
+   *
+   * **NOTE**: All nodes that are up to and including `depthInHierarchy` will be expanded. Node at the `depthInHierarchy` position will be expanded.
```

Comment:
I think the second sentence is repeating the first one.

```suggestion
   * **NOTE**: All nodes that are up to and including `depthInHierarchy` will be expanded.
```